blocks/mbedActions.js

```diff
@@ -657,18 +657,31 @@ Blockly.Blocks['mbedActions_pin_set_pull'] = {
         this.setColour(Blockly.CAT_ACTION_RGB);
         var pull = new Blockly.FieldDropdown(
                 [ [ Blockly.Msg.PIN_PULL_UP, 'UP' ], [ Blockly.Msg.PIN_PULL_DOWN, 'DOWN' ], [ Blockly.Msg.PIN_PULL_NONE, 'NONE' ] ]);
-        this.dropDownPorts = getConfigPorts('digitalin');
+        this.dropDownPorts = getConfigPorts('');
```

Comment:
Don't we need here 'digitalout' config ports?

Reply:
I removed this so if the 'mbed_pin_set_pull' block is present in a program, it's not possible to change the pin.